ADD  -- Task 15(G): Creación de dashboard para Admin.

alesanfe opened the pull request.

alesanfe committed:
feat(init): Admin dashboard creation

A class has been created which will contain values that have been previously calculated on a temporary basis.

alesanfe committed:
feat(updated): Admin dashboard creation

The attribute to count each of the accounts for a given role has been separated into multiple attributes for ease of use.

alesanfe committed:
fix(update): Admin dashboard creation

One of the attributes could be derived from criticalBulletinsRatio.

alesanfe committed:
fix(update): Admin dashboard creation

The statistics values cannot be negative.

marvicmar reviewed (changes requested):
Eliminar message de la restricciones

alepervaz reviewed (changes requested):
eliminar mensajes

alesanfe committed:
fix(update): Admin dashboard creation

Custom messages have been removed from the restrictions to keep the application in both English and Spanish.

marvicmar reviewed (approved):
El código esta correcto y completo

alepervaz reviewed (approved):
Todo correcto
